File: w06/d01/jimmy_davis/js/calculator.js
var calculator = {
  add: function(a, b) {
    return a + b
  },
  subtract: function(a, b) {
    return a - b
  },
  multiply: function(a, b) {
    return a * b
  },
  divide: function(a, b) {
    return a / b
  },
  modulus: function(a, b) {
    return a % b
  },

  calculate: function() {
    a = parseInt(prompt("Enter first number"), 10);
    b = parseInt(prompt("Enter second number"), 10);
    operator = prompt("Enter operator (+, -, *, /, or %)");
    switch(operator) {
      case "+":
        console.log(this.add(a, b));
        break;
      case "-":
        console.log(this.subtract(a, b));
        break;
      case "*":
        console.log(this.multiply(a, b));
        break;
      case "/":
        console.log(this.divide(a, b));
        break;
      case "%":
        console.log(this.modulus(a, b));
        break;
    }
  }
};

calculator.calculate();

File: w01/d02/katherine_santiago/calculator.rb

while (num1 != "q") || (num2 != "q") || (operation != "q")

def menu
  puts "CALCULATOR"
  puts "Balance = 0"
  puts " + - * /"
  puts "Enter first number or enter 'q' to quit"
  num1 = gets.downcase.chomp!
end

if num1 == "q"
  puts "Thanks, bye!"

elsif
  num1 = num1.to_f
  puts "type '+' to add"
  puts "type '-' to substract"
  puts "type '*' to multiply"
  puts "type '/' to divide"
  puts "What would you like to do?"
end

calculator = gets.chomp!
if calculator != '+' && calculator != '-' && calculator != '*' && calculator  != '/'
    puts "Please enter '+' '-' '*' or '/'"
    calculator = gets.chomp!
else
    puts "Enter second number"
num2 = gets.chomp!
num2 = num2.to_f
end


if calculator == '+'
  ans = num1 + num2
  puts "Answer = #{ans}"

elsif calculator == '-'
  ans = num1 - num2
  puts "Answer = #{ans}"

elsif calculator == '*'
  ans = num1 * num2
  puts "Answer = #{ans}"

elsif calculator == '/'
  ans = num1 / num2
  puts "Answer = #{ans}"

end

end

end








